Tests/ClientTests/Frontend/Home/FirefoxHomeJumpBackInViewModelTests.swift

```diff
@@ -287,24 +326,17 @@ class FirefoxHomeJumpBackInViewModelTests: XCTestCase {
     }
 
     func test_updateData_mostRecentTab_multipleDesktopClients() {
-        let profile = MockProfile()
         let remoteClient = remoteDesktopClient(name: "Fake Client 2")
         let remoteClientTabs = remoteTabs(idRange: 7...9)
-        profile.mockClientAndTabs = [ClientAndTabs(client: remoteDesktopClient(), tabs: remoteTabs(idRange: 1...5)),
+        mockProfile.mockClientAndTabs = [ClientAndTabs(client: remoteDesktopClient(), tabs: remoteTabs(idRange: 1...5)),
                                      ClientAndTabs(client: remoteClient, tabs: remoteClientTabs)]
-        subject = JumpBackInViewModel(
-            isZeroSearch: false,
-            profile: profile,
-            isPrivate: false,
-            tabManager: mockTabManager
-        )
 
         let expectation = XCTestExpectation(description: "Main queue fires; updateRemoteTabs(completion:) is called.")
         subject.updateData {
             expectation.fulfill()
         }
 
-        wait(for: [expectation], timeout: 5.0)
+        wait(for: [expectation], timeout: 15.0)
```

Comment:
What did the timeout needed to be increased? If we're using `mockClientAndTabs` 🤔

Reply:
Tests were failing in the past and the higher timeout solved it. Lowered it again as it doesn't seem to be an issue anymore.